week-2/L-array-methods-2/exercise2.js

```diff
@@ -7,7 +7,11 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
+function capitalise(str) {
+  var str0 = str.slice(1);
+  var str1 = str.slice(0, 1).toUpperCase();
+  return str1.concat(str0);
```

Comment:
Good. For future reference you can do `str.charAt(0)` instead of `str.slice(0,1)`, and it's very slightly more efficient.